phpdotnet/phd/Package/PHP/XHTML.php

```diff
@@ -69,8 +69,12 @@ abstract class Package_PHP_XHTML extends Package_Generic_XHTML {
             ),
         ),
         'type'                  => array(
-            /* DEFAULT */          'span',
-            'methodsynopsis'    => 'format_suppressed_tags',
+            /* DEFAULT */          'format_type',
+            'methodsynopsis'    => 'format_methodsynopsis_type',
+            'type'              => array(
+                /* DEFAULT */       'format_type',
+                'methodsynopsis' => 'format_methodsynopsis_type',
+            ),
```

Comment:
I just noticed that the distinction between `format_type()` and `format_methodsynopsis_type()` is useless (both are doing the same). I'll fix this later today.

Reply:
Nope, nonsense.